backend/Terraform/main.tf

```diff
@@ -1,3 +1,20 @@
-module "lambda" {
-    source = "./modules"
-}
\ No newline at end of file
+# Define S3 bucket object (source code) for Lambda function
+data "aws_s3_bucket_object" "lambda_zip_gda" {
+  bucket = var.s3_bucket
+  key    = "lamda_functions/google_drive_analytics.zip" #all functions will be stored in the lamda_functions folder inside the bucket
```

Comment:
The functions will be uploaded by CI/CD to that location?

Reply:
according to the ci workflow it is ....